### ISSUE TITLE
Error then import { Store } from 'svelte/store.js'; 

### ISSUE THREAD

**Issue by mahadoang** [meteor-svelte/meteor-svelte#20]
import { Store } from 'svelte/store.js'; 

Uncaught (in promise) SyntaxError: Unexpected token {
    at eval (modules.js?hash=e85cfce550754a19536a1fda19bffb3b178b0c60:26034)

**Comment by klaussner** [comment]
This error is thrown because `svelte/store.js` contains an `import` statement. The easiest way to make it work with Meteor is to import Svelte's pre-compiled version `svelte/store.umd.js` instead of `svelte/store.js`. However, if you want to continue using `svelte/store.js`, you have to tell Meteor that it should compile the two files `svelte/store.js` and `svelte/shared.js` like all your other JavaScript modules. You can do this by creating a symlink to these files in your application directory, for example:

```sh
cd imports
ln -s ../node_modules/svelte/shared.js
ln -s ../node_modules/svelte/store.js
```

For more details, check out this forum post (it uses the Polymer library as an example): https://forums.meteor.com/t/litelement-import-litelement-html/45042/8.

Let me know if either way works for you. :slightly_smiling_face:

**Comment by mahadoang** [comment]
Thanks. @klaussner 
Not works for me.

Display error

`Master.html:2 Uncaught (in promise) Error: <Master> references store properties, but no store was provided`

Master.html
```
<NavContainer />
<div id="container-main">{$name}</div>
<script>
    import NavContainer from '../components/NavContainer';
    import roadtrip from 'roadtrip';
    import store from '../store.js';
    export default {
        store: () => store,
        components:{
			NavContainer
		}
    }
</script>
```
store.js

```
import { Store } from 'svelte/store';
export default new Store({ name: 'world' });
```

**Comment by klaussner** [comment]
This turned out to be a bug in Svelte, which was fixed in version 2.15.1. I published a new version of `svelte:compiler` with the latest Svelte version (2.15.3). If you update with `meteor update svelte:compiler`, the store should work.

**Comment by mahadoang** [comment]
Thanks. @klaussner 
it works for me.
